tests/stress_tests/scripts/compare_memcheck_2_runs.py

```diff
@@ -181,6 +186,6 @@ def cli_parser():
 
 if __name__ == "__main__":
     args = cli_parser()
-    references = get_memcheck_records(args.ref_source, args.db_collection, DATABASE, args.db_url)
+    references = get_memcheck_records(args.ref_source, args.ref_db_collection, DATABASE, args.db_url)
```

Comment:
Can we use db_collection here if ref_db_collection is not defined? This will simplify the command line.

Reply:
I don't want to mix the logic applied for current and reference sources. In case of using more than one reference, it will be hard to extend the logic in a right way, so I want to not touch `--db_collection` at all. Also in run_memcheck.py there are a lot of logic within `db_collection`, which will be hard to handle correctly (or it won't be aligned with compare_memcheck_2_runs.py)